test/JDBC/input/ownership/sys_server_role_members-vu-verify.sql

```diff
@@ -0,0 +1,32 @@
+SELECT * from sys_server_role_members_vu_prepare_view;
+GO
+
+ALTER SERVER ROLE sysadmin ADD MEMBER sys_server_role_members_vu_prepare_login1;
+GO
+
+ALTER SERVER ROLE sysadmin ADD MEMBER sys_server_role_members_vu_prepare_login2;
+GO
+
+SELECT * from sys_server_role_members_vu_prepare_view;
+GO
+
+ALTER SERVER ROLE sysadmin DROP MEMBER sys_server_role_members_vu_prepare_login2;
+GO
+
+ALTER SERVER ROLE sysadmin ADD MEMBER sys_server_role_members_vu_prepare_login3;
+GO
```

Comment:
Here, we need to test visibility according to the current_login. There may be a case that this view can only show data about current login, not others unless it has some sysadmin - we need to verify such scenarios.

Please check the .mix files for other test - We can also use multiple connections with specific login & database.